libraries/adaptive-expressions/src/builtinFunctions/titleCase.ts

```diff
@@ -22,17 +24,24 @@ export class TitleCase extends StringTransformEvaluator {
     }
 
     /**
+     * @param args
+     * @param options
      * @private
      */
```

Comment:
Our lint rules should be fixed such that private methods don't need any JSDoc strings. IMO it's better to remove them.

Reply:
removed